.changeset/fresh-chairs-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-cookie-banner": minor
+---
+
+[Added] - Follow up change requests from previous pr
```

Comment:
I would suggest writing something that describes the actual changes being made rather than alluding to PR change requests. Otherwise it makes it very difficult to understand what was changed when we are reviewing the changelog (especially for consumers of our packages)

Reply:
(and to nitpick, I think it could be `Changed` rather than `Added`)

Reply:
I think it should be added as i have added tests, a new prop to the readme etc. No actual functional changes: https://keepachangelog.com/en/1.1.0/

Reply:
Perhaps no changeset file should be added in that case? Because the current message doesn't indicate no functional changes were added